api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -673,6 +669,7 @@ public void updateGlobalProperty_shouldFailIfUserIsNotAllowedToEditGlobalPropert
 	public void updateGlobalProperty_shouldUpdateIfUserIsAllowedToEditGlobalProperty() {
 		executeDataSet(ADMIN_INITIAL_DATA_XML);
 		GlobalProperty property = getGlobalPropertyWithEditPrivilege();
+		GlobalProperty globalPropertyWithViewPrivilege = getGlobalPropertyWithViewPrivilege();
```

Comment:
Why get this from here when we are going to use it way below. It would be great to fetch it closes to the point of usage. What do you think?

Reply:
This allows us to call the getGlobalPropertyWithViewPrivilege method without adding the proxy privilege to the user. Otherwise we'll have to add proxy privilege before calling this method.